client/__tests__/e2e/e2eAnnotations.test.ts

```diff
@@ -551,65 +364,4 @@ describe.each([
 
     expect(result).toMatchSnapshot();
   });
-
-  test("truncate midpoint whitespace", async () => {
```

Comment:
@seve Is this something that needs to be tested? I don't think labels can be renamed. Is there automatic whitespace truncation when loading annotations into the frontend? If so, I'll add the test back.

Reply:
see https://github.com/chanzuckerberg/cellxgene/issues/1578